test/Directory.Build.targets

```diff
@@ -1,5 +1,14 @@
 <Project>
 
   <Import Project="..\Directory.Build.targets" />
-
+  
+  <PropertyGroup Condition="'$(Coverage)' == 'true'">
+      <CollectCoverage>true</CollectCoverage>
+      <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+      <CoverletOutputFormat>opencover</CoverletOutputFormat>
+      <CoverletOutput>$(MSBuildThisFileDirectory)..\bin\coverage\$(AssemblyName).xml</CoverletOutput>
+      <Include>[Microsoft.ML.*]*</Include>
+      <ExcludeByFile>$(MSBuildThisFileDirectory)**\*.*</ExcludeByFile>
```

Comment:
💡 Coverage in test files is typically a good thing, for the same overall reasons why coverage in production code is good.

```suggestion
```